CHANGELOG.md

```diff
@@ -6,6 +6,11 @@
 
 #### New Features
 
+#### Improvements
+- Disables sql simplification when sort is performed after limit. 
```

Comment:
Are following details needed to be known by users publicly? Or Snowflake compiler details? If not, maybe we don't need to mention them?

Reply:
For this one, yes. This is not snowflake compiler details, more of snowpark generated query behavior. Event hough we haven't provide any guarantee, we still want to notify the user about this change, in case this is causing any problem to there workloads